.github/workflows/chapter-4-contracts-workflow.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         with:
           dotnet-version: 8.0.x
       - name: Add Evolutionary Architecture Nuget Source
-        uses: evolutionary-architecture/evolutionary-architecture-by-example/.github@main
+        uses: evolutionary-architecture/evolutionary-architecture-by-example/.github@feature/add_signature_value_object
```

Comment:
Please change it back to main (I assume this is the leftover from testing)

Reply:
This is not for testing. I have to merge it beacuse this MR contains changes in action. After merge this PR to main. I'm going to revert.